ONSdigital/dp-frontend-search-controller
# Remove logic to switch between ES 7.10 API response and ES 2.2 API response

### What

See title, but also includes ripping out departments from controller

### How to review

- check changes make sense
- check unit tests pass
- run search services locally to see controller responding correctly, services needed:
  - dp-search-api (I actually tunnelled into sandbox environment and used port forwarding)
  - dp-api-router
  - dp-topics-api
  - zebedee (homepage content needed, otherwise page will return 500)

### Who can review

Anyone
